mkosi.md

```diff
@@ -524,7 +533,11 @@ details see the table below.
   invoked. By default, the build script runs with networking turned
   off. The `$WITH_NETWORK` environment variable is passed to the
   `mkosi.build` build script indicating whether the build is done with
-  or without this option.
+  or without this option. If specified as `--with-network=strict` the
+  package manager is instructed not to contact the network for
+  updating package data. This provides a minimal level of
+  reproducibility, as long as the package data cache is already fully
+  populated.
```

Comment:
The naming of this option seems rather strange: the default is no network, `--with-network` enables network, `--with-network=strict` does the opposite. "strict" is meaningless. Maybe call with `--with-network=never` instead?